lib/run.js

```diff
@@ -39,7 +39,6 @@ function eslint() {
 
   const cli = new eslint.CLIEngine({
     extensions: ['.js', '.jsx', '.ts', '.tsx'],
-    ignorePath: '.gitignore'
```

Comment:
An ignore file is an important part of this action for me. Otherwise, ESLint will attempt to lint `node_modules` for some projects.

I'd suggest something like

```js
ignorePath: SOURCE_ROOT ? null : '.gitignore'
```

until I can figure out a better way to include ignore files in the inputs to the action